pkg/gadgets/run/types/metadata.go

```diff
@@ -36,6 +36,9 @@ const (
 	// Prefix used to mark snapshotters structs
 	snapshottersPrefix = "gadget_snapshotter_"
 
+	// Prefix used to mark mapIters info
+	mapIterInfoPrefix = "gadget_mapiter_"
+
```

Comment:
I'm not sure we want to add this commit?

Reply:
You already implemented it, it works fine, so let's add it.